packages/components/src/textarea-control/README.md

```diff
@@ -14,7 +14,7 @@ TextareaControls are TextControls that allow for multiple lines of text, and wra
 
 ### Usage
 
-**When to use TextareaControl**
+#### When to use **TextareaControl**
```

Comment:
I wonder if the formatting here for the TextareaControl bit should actually be a code block, like…

```md
#### When to use `TextareaControl`
```

Reply:
(Related, I wonder if the Gutenberg handbook docs will format `h4` level paragraphs as bold anyway, making the extra bolding redundant?)

Reply:
> (Related, I wonder if the Gutenberg handbook docs will format h4 level paragraphs as bold anyway, making the extra bolding redundant?)

I just looked this up, and it appears that the extra bold tags aren't necessary — the handbook (and GitHub) will show the entire H4 as bold. 

I could go either way about the code blocks — on one hand, it technically makes sense, but on the other hand, it'd require a bit of cleanup and enforcement going forward... the easier it is to update the handbook, the better. 🙂

Reply:
Makes sense to me, @kjellr. Probably fine to format without the code blocks :)

Reply:
Updated!